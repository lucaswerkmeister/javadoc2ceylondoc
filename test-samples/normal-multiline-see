/**
 * line 1
 *
 * @see java.lang.String
 * @see File.Appender#write()
 * @see java.lang.Integer#parseInt(String)
 * @see String#replace(CharSequence, CharSequence)
 * @see MySingleton.instance
 */
