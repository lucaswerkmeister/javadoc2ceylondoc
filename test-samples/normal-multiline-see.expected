"line 1"
see(`class String`)
see(`function File.Appender.write`)
see(`function Integer.parseInt`)
see(`function String.replace`)
see(`value MySingleton.instance`)
